modules/commands/list.json

```diff
@@ -500,6 +518,56 @@
       }
     ]
   },
+  {
+    "name": "ssl",
+    "aliases": [
+      "usessl"
+    ],
+    "title": "The useSSL error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
+    "fields": [
+      {
+        "key": "Read me!",
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"

```

Comment:
Don't think it's necessary to link this.

Plus it talks about SSL mainly in the context of HTTPS, which is not relevant here.

Reply:
removed